wled00/wled.cpp

```diff
@@ -338,6 +338,7 @@ void WLED::setup()
 
   #ifdef WLED_ENABLE_ADALIGHT
   if (!pinManager.isPinAllocated(3)) {
+    // TODO: Should this be reserved for Ada?  Serial input pin?  Else, doesn't appear in pin ownership list....
```

Comment:
GPIO3 is a problem, but flat out reserving it would be overkill. Adalight and serial JSON will only be used by a few, and these users would _maybe_ run into issues if they configured GPIO3 to serve other purposes.

GPIO3 is valuable for two main reasons, it is one of only 3 non-bitbang outputs on ESP8266, and it is used as an LED output on all QuinLED boards.

Reply:
I have circumvented this inconvenience by defaulting to not build with ADA Light option.
You have to explicitely enable ADA.

Reply:
OK, added to attempt allocation of pins 1 & 3 to adalight.